pkg/chartutil/capabilities.go

```diff
@@ -92,13 +79,56 @@ func allKnownVersions() VersionSet {
 	// We should register the built in extension APIs as well so CRDs are
 	// supported in the default version set. This has caused problems with `helm
 	// template` in the past, so let's be safe
-	apiextensionsv1beta1.AddToScheme(scheme.Scheme)
-	apiextensionsv1.AddToScheme(scheme.Scheme)
+	s := runtime.NewScheme()
```

Comment:
So one of the main reasons why we were using the upstream variable was to ensure we picked up new schemas as they became available in future Kubernetes versions.

This now looks like we're attempting to reconstruct our own schema, but that comes with the risk that should a new schema comes up, it will not be added.

Perhaps a better approach might be to propose a new function upstream that will recreate `scheme.Scheme` exactly as-is so that we don't need to maintain all of this cruft to rebuild the same schema from scratch.